lib/central_mail/utilities.rb

```diff
@@ -29,10 +29,9 @@ def log_submission(uploaded_object, metadata)
       pdf_total = number_pages.count
 
       title = uploaded_object.class.to_s.split('::').first
+
       Rails.logger.info("#{title}: Submission success",
-                        'uuid' => metadata['uuid'],
-                        'source' => metadata['source'],
-                        'docType' => metadata['docType'],
+                        'metadata' => metadata,
```

Comment:
we could just pass metadata.merge({consumer_id: uploaded_object.consumer_id, ...}) into Rails.logger.info. That way all the keys will still be there and the extra data can be inserted at the same time, rather than specifically outputting the keys & values individually.

Reply:
yeah, not going to do it individually. i just broke my pr. hold please.